### ISSUE TITLE
Error with property > P4658 with claim update

### ISSUE THREAD

**Issue by Manu1400** [maxlath/wikidata-edit#15]
If you do `wdEdit.claim.update(qid, 'P4700', initialValue, newValue)`
I have this error :

```
Unhandled rejection TypeError: Cannot read property 'type' of undefined
    at module.exports.propertyId (/Users/emmanuel/node_modules/wikidata-edit/lib/properties/find_datatype.js:24:55)
    at module.exports (/Users/emmanuel/node_modules/wikidata-edit/lib/claim/find_snak.js:7:20)
    at getEntityClaims.then.claims (/Users/emmanuel/node_modules/wikidata-edit/lib/claim/update.js:21:19)
    at tryCatcher (/Users/emmanuel/node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/Users/emmanuel/node_modules/bluebird/js/release/promise.js:512:31)
    at Promise._settlePromise (/Users/emmanuel/node_modules/bluebird/js/release/promise.js:569:18)
    at Promise._settlePromise0 (/Users/emmanuel/node_modules/bluebird/js/release/promise.js:614:10)
    at Promise._settlePromises (/Users/emmanuel/node_modules/bluebird/js/release/promise.js:693:18)
    at Async._drainQueue (/Users/emmanuel/node_modules/bluebird/js/release/async.js:133:16)
    at Async._drainQueues (/Users/emmanuel/node_modules/bluebird/js/release/async.js:143:10)
    at Immediate.Async.drainQueues (/Users/emmanuel/node_modules/bluebird/js/release/async.js:17:14)
    at runCallback (timers.js:781:20)
    at tryOnImmediate (timers.js:743:5)
    at processImmediate [as _immediateCallback] (timers.js:714:5)
```

The issue is only with property > P4658.

When i add  `'P4700': { 'type': 'ExternalId', 'label': 'Cairn journal ID' },` to properties.js, the bug is fixed.

Please update the list of wikidata properties.

**Comment by Manu1400** [comment]
Note: in struct `{ 'type': 'ExternalId', 'label': 'Cairn journal ID' }` the label value is never used.

**Comment by maxlath** [comment]
update published as `v2.0.11`

**Comment by maxlath** [comment]
5fac55d should help to produce clearer error messages
